### PR TITLE
fixed issue with nested messages and enums

### PR DESCRIPTION
**Pull request by Vadim1024**

### CODE REVIEW

**Comment by hchasestevens** [comment]
Hi @Vadim1024 , this looks great - would you mind:
* Adding a new schema with nested enum and message types (as well as its compiled python file) to https://github.com/CurataEng/hypothesis-protobuf/tree/master/tests/unit/test_schemas
* Adding unit tests to https://github.com/CurataEng/hypothesis-protobuf/blob/master/tests/unit/test_module_conversion.py demonstrating that we can successfully produce messages for that schema's generated strategies, including both the parent and nested child strategies
* Bumping the version number in https://github.com/CurataEng/hypothesis-protobuf/blob/master/setup.py to 1.1.0 (for release to PyPI).

File: tests/unit/test_module_conversion.py [hunk 1]
```diff
@@ -23,6 +23,10 @@ def test_instant_message_example():
     assert isinstance(instant_message_example.recipient.screen_name, basestring)
     assert isinstance(instant_message_example.message, basestring)
     assert isinstance(instant_message_example.metadata.latency, float)
+    assert isinstance(instant_message_example.metadata.inner.a, float)
+    assert isinstance(instant_message_example.metadata.inner.layer.client.name, basestring)
+    assert isinstance(instant_message_example.metadata.inner.layer.status, Number)
+    assert isinstance(instant_message_example.client, Number)
```
Comment (hchasestevens) [inline tests/unit/test_module_conversion.py:29]: 👍 

Would you also mind adding just one more test? The following would be fine:
```python
def test_nested_strategies_produce_data():
    """Ensure nested messages are accessible within strategy dict."""
    protobuf_strategies = modules_to_strategies(im_pb2)
    assert protobuf_strategies[im_pb2.MetaData.Inner].example()
    assert protobuf_strategies[im_pb2.MetaData.Inner.LimboDreamLayer].example()
```

**Review by hchasestevens — APPROVED** [review]
Looks great - one minor comment, but otherwise looks to pass the test suite:

```
hchasestevens@CHASE-DESKTOP:/mnt/c/Users/chase/Documents/GitHub/hypothesis-protobuf$ tox .
GLOB sdist-make: /mnt/c/Users/chase/Documents/GitHub/hypothesis-protobuf/setup.py
py27 inst-nodeps: /mnt/c/Users/chase/Documents/GitHub/hypothesis-protobuf/.tox/dist/hypothesis-pb-1.1.0.zip
py27 installed: attrs==17.4.0,coverage==4.5.1,enum34==1.1.6,funcsigs==1.0.2,future==0.16.0,hypothesis==3.48.1,hypothesis-pb==1.1.0,pluggy==0.6.0,protobuf==3.5.2,py==1.5.2,pytest==3.4.2,six==1.11.0
py27 runtests: PYTHONHASHSEED='3895636698'
py27 runtests: commands[0] | py.test ./tests
========================================= test session starts =========================================
platform linux2 -- Python 2.7.12, pytest-3.4.2, py-1.5.2, pluggy-0.6.0
rootdir: /mnt/c/Users/chase/Documents/GitHub/hypothesis-protobuf, inifile:
plugins: hypothesis-3.48.1
collected 2 items

tests/unit/test_module_conversion.py ..                                                         [100%]

====================================== 2 passed in 0.21 seconds =======================================
py36 inst-nodeps: /mnt/c/Users/chase/Documents/GitHub/hypothesis-protobuf/.tox/dist/hypothesis-pb-1.1.0.zip
py36 installed: attrs==17.4.0,coverage==4.5.1,future==0.16.0,hypothesis==3.48.1,hypothesis-pb==1.1.0,pluggy==0.6.0,protobuf==3.5.2,py==1.5.2,pytest==3.4.2,six==1.11.0
py36 runtests: PYTHONHASHSEED='3895636698'
py36 runtests: commands[0] | py.test ./tests
========================================= test session starts =========================================
platform linux -- Python 3.6.3, pytest-3.4.2, py-1.5.2, pluggy-0.6.0
rootdir: /mnt/c/Users/chase/Documents/GitHub/hypothesis-protobuf, inifile:
plugins: hypothesis-3.48.1
collected 2 items

tests/unit/test_module_conversion.py ..                                                         [100%]

====================================== 2 passed in 0.15 seconds =======================================
_______________________________________________ summary _______________________________________________
  py27: commands succeeded
  py36: commands succeeded
  congratulations :)
```

@Vadim1024 Thank you very much for your contribution.

File: tests/unit/test_module_conversion.py (hunk 1 above)
Comment (Vadim1024) [reply tests/unit/test_module_conversion.py:29]: Done :)

**Comment by hchasestevens** [comment]
Hello @Vadim1024 , just updating to let you know I have released version 1.1.0 (with your changes) to PyPI.

**Pull request merged**